release/_jira.sh

```diff
@@ -32,29 +32,7 @@ function add_jira_issue {
 }
 
 function add_jira_issue_comment {
-	local data=$(
-		cat <<- END
-		{
-			"body": {
-				"content": [
-					{
-						"content": [
-							{
-								"text": "${1}",
-								"type": "text"
-							}
-						],
-						"type": "paragraph"
-					}
-				],
-				"type": "doc",
-				"version": 1
-			}
-		}
-		END
-	)
```

Comment:
@lucasmiranda0, instead of moving the JSON creation out of this function, can we keep it here and, alternatively, create a new function named `add_jira_issue_comment_with_mention` to support your need?